### PR TITLE
Clarify layoutSizeDefined and provide default isLayoutSupported

### PR DESCRIPTION
**Pull request by dvoytenko**
Not a "problem solved" yet. But I think this is a step forward in clarifying things. Also, I think, @alanorozco, you wanted to do this anyway?

### CODE REVIEW

File: src/preact/base-element.js [hunk 1]
```diff
@@ -541,6 +551,13 @@ export class PreactBaseElement extends AMP.BaseElement {
 
 // Ideally, these would be Static Class Fields. But Closure can't even.
 
+/**
+ * An experiment required for this component.
+ *
+ * @protected {string}
+ */
+PreactBaseElement['experiment'] = '';
+
```
Comment (alanorozco) [inline src/preact/base-element.js:560]: Is this implemented?

**Review by alanorozco — APPROVED** [review]
RIP #30260

**Comment by dvoytenko** [comment]
> RIP #30260

Oops, didn't realize you already have one...

File: src/preact/base-element.js (hunk 1 above)
Comment (dvoytenko) [reply src/preact/base-element.js:560]: Oops. Forgot to cleanup. I tried this in one of the iterations. But realized that the `isExperimentOn` requires strict constants (which is a good idea). So I dropped this idea for now.

**Pull request merged**